src/common/metrics/hospitalizations.tsx

```diff
@@ -107,28 +119,14 @@ function renderStatus(projections: Projections): React.ReactElement {
   }
 
   const totalICUBeds = formatInteger(icu.totalBeds);
-  const nonCovidUsedBeds = formatInteger(icu.nonCovidPatients);
-  const nonCovidUsedBedsPercent = formatPercent(
-    icu.nonCovidPatients / icu.totalBeds,
-  );
-  const remainingICUBeds = formatInteger(icu.totalBeds - icu.nonCovidPatients);
+  const nonCovidICUPatients = formatInteger(icu.nonCovidPatients);
   const covidICUPatients = formatInteger(icu.covidPatients);
+  const totalICUPatients = formatInteger(
+    icu.nonCovidPatients + icu.covidPatients,
```

Comment:
This seems fine because we explicitly break it out in the icu headroom metric - maybe in the future it would be better to just use the `actuals.icuBeds.currentUsageTotal` so we're not relying on the icu headroom code?

Reply:
Yep, will do in a subsequent PR!